SPEC.md

```diff
@@ -76,22 +94,33 @@ $ ipfs files rm  /cohosting/docs.ipfs.io
    /ipfs/Qmd41WqbCsfTx4wJvP6vvv3hHb46bEHG1hC6Kqt7mhGQUR
    ```
 
-4. Ensure contents are in the local repo (below command will fetch missing blocks in the background)
+4. **If** it is a **fully** cohosted website, ensure contents are in the local repo (below command will fetch missing blocks in the background)
```

Comment:
`ipfs refs --recursive` is a blocking operation, but it can take time, so may be a good idea  to run it in the background, removing problem in automated situations when new `sync` call is running before the old one finished.

I think we could leave decision to the implementer:

```suggestion
4. **If** it is a **fully** cohosted website, ensure contents are in the local repo:
```